api/src/pcapi/alembic/versions/20250226T165636_50fe0db503c3_add_is_active_to_national_program.py

```diff
@@ -0,0 +1,24 @@
+"""
+Add isActive to national_program
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# pre/post deployment: pre
+# revision identifiers, used by Alembic.
+revision = "50fe0db503c3"
+down_revision = "9d1b6d3edf16"
+branch_labels: tuple[str] | None = None
+depends_on: list[str] | None = None
+
+
+def upgrade() -> None:
+    op.add_column(
+        "national_program", sa.Column("isActive", sa.Boolean(), server_default=sa.text("true"), nullable=False)
```

Comment:
Detail : ceci devrait marcher

```suggestion
        "national_program", sa.Column("isActive", sa.Boolean(), server_default=sa.true(), nullable=False)
```

Reply:
J'ai laissé faire l'autogenerate

Il y a 2 autres exemples dans les migrations de colonnes isActive, qui ont aussi sa.text

Je serais d'avis de laisser comme ça